source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,306 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn how to query **geospatial data**
+with the MongoDB Java Driver. 
+
+Geospatial data is data tied to a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with the **GeoJSON**
+format. GeoJSON is a geospatial data format. You can use the GeoJSON format to
+store your data, and you must use the GeoJSON format to query your geospatial
+data with MongoDB. 
```

Comment:
This paragraph feels a bit fragmented.

1. You can represent `geospatial data` on an Earth-like sphere with the `**GeoJSON** format`.
2. GeoJSON is a geospatial data format.
The only words that don't overlap are "is a". 

You can use [the] GeoJSON format to store your data, and ... This doesn't feel very prescriptive. Why not something like
> Store and query your geospatial data in MongoDB using the GeoJSON format.

Reply:
Thanks for the note. I agree that the first two sentence in this section are clunky and should be changed. I removed "format" from the first sentence, and let the second sentence do this work. I now introduce "IETF" in the second sentence.

Fix:

In regards to the second comment, I feel that "you can" ([based off of the Style Guide's definition of "can"](https://docs.mongodb.com/meta/style-guide/writing/use-effective-verbs/#use-modal-verbs-accurately))is appropriate as the page does not advise whether you should store your data as GeoJSON or Legacy coordinate pairs. By saying "store and query your data...", I believe we are implicitly stating that you should not use Legacy Coordinate Pairs (and if readers should not use legacy coordinate pairs than legacy coordinate pairs should be marked as deprecated).